### PR TITLE
TEAMNADO-4579 - Update packages

### PR DESCRIPTION
**Pull request by BrooklynLCodes**

### CODE REVIEW

**Review by jlgre — COMMENTED** [review]
This doesn't run or compile successfully right now - I think its because you are importing `InventoryRoutes` which doesn't exist. That should probably be `ManifestRoutes`

**Comment by BrooklynLCodes** [comment]
> This doesn't run or compile successfully right now - I think its because you are importing `InventoryRoutes` which doesn't exist. That should probably be `ManifestRoutes`

I did fix this issue, i got it to run. I believe this is resolved.

**Comment by jlgre** [comment]
> > This doesn't run or compile successfully right now - I think its because you are importing `InventoryRoutes` which doesn't exist. That should probably be `ManifestRoutes`
> 
> I did fix this issue, i got it to run. I believe this is resolved.

That does look fixed - I am still running into some errors when trying to open the application though. I get this whenever I navigate to manifests.

`Uncaught Error: Objects are not valid as a React child (found: object with keys {$$typeof, _payload, _init}). If you meant to render a collection of children, use an array instead.`

File: package.json [hunk 1]
```diff
@@ -93,74 +93,74 @@
...
     "@babel/preset-flow": "7.18.6",
     "@babel/preset-react": "7.18.6",
     "@babel/preset-typescript": "^7.18.6",
     "@faker-js/faker": "^7.5.0",
     "@redhat-cloud-services/eslint-config-redhat-cloud-services": "^1.2.4",
-    "@redhat-cloud-services/frontend-components-config": "^4.6.21",
+    "@redhat-cloud-services/frontend-components-config": "^4.7.0",
     "@testing-library/jest-dom": "^5.16.5",
-    "@testing-library/react": "^12.1.2",
+    "@testing-library/react": "^12.0.0",
```
Comment (jlgre) [inline package.json:118]: I would leave this line how it was. Both will install 12.x.x on `npm i` but the new change will allow older versions of this package to be supported.

File: package.json [hunk 2]
```diff
@@ -93,74 +93,74 @@
...
     "@testing-library/react-hooks": "^8.0.1",
-    "@types/jest": "^29.1.2",
-    "@types/js-cookie": "^3.0.2",
-    "@types/lodash": "^4.14.186",
-    "@types/react": "^17.0.19",
+    "@types/jest": "^29.4.0",
+    "@types/js-cookie": "^3.0.3",
+    "@types/lodash": "^4.14.191",
+    "@types/react": "^17.0.9",
```
Comment (jlgre) [inline package.json:123]: Same comment here

Comment (jlgre) [reply package.json:123]: Specifically on `@types/react`

**Review by jlgre — APPROVED** [review]
Looks good!

**Pull request merged**